extras/hpa/frontend.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START gke_boa_kubernetes_manifests_frontend_horizontal_pod_autoscaler_frontend]
+apiVersion: autoscaling/v2beta2
+kind: HorizontalPodAutoscaler
```

Comment:
Could you add a README.md file in this new directory explaining what these HPAs are for, how to deploy it (should be simple -- "go to this directory and kubectl apply..."), to add more context to those ending up here.